spec/spec_helper.rb

```diff
@@ -25,6 +25,9 @@ def log_in_user(user)
 end
 
 def create_and_log_in_user(user_attrs = {})
+  defaults = { email: "test@example.com", role: "challenge_manager" }
+  user_attrs = defaults.merge(user_attrs)
```

Comment:
can you move these defaults into the `create_user` function, since that's ultimately where they take affect